java-plugin/src/test/files/UnnecessarilyAssignValuesToVariablesTestCheck.java

```diff
@@ -92,4 +92,13 @@ public void testNonCompliantThrow() throws Exception {
     public void testCompliantThrow() throws Exception {
         throw new Exception("dummy");
 	}
-}
\ No newline at end of file
+
+    public void testCompliantCatchException() throws Exception {
```

Comment:
ok for me, but, please remove "throws Exception" because no Exception really thrown.

Reply:
@natixis-caen : last modification , then I will approve this PR :p